### PR TITLE
Chrisk bugfixes

### PR DESCRIPTION
**Pull request by chriskr**

### CODE REVIEW

File: src/ecma-debugger/js-source-view.js [hunk 1]
```diff
@@ -1141,7 +1141,7 @@ cls.JsSourceView.create_ui_widgets = function()
   );
 
   var service = window.services['ecmascript-debugger'];
-  var stop_on_error = service.satisfies_version(6, 8) ? 1 : 0;
+  var stop_on_error = service.satisfies_version(6, 8) ? true : false;
```
Comment (hzr) [inline src/ecma-debugger/js-source-view.js]: `service.satisfies_version()` returns a boolean.

File: src/ui-strings/ui_strings-en.js [hunk 2]
```diff
@@ -456,6 +471,9 @@ ui_strings.S_BUTTON_LABEL_HIDE_DEFAULT_PROPS_IN_GLOBAL_SCOPE = "Show default pro
 /* DESC: List item under the Source settings menu to logs all threads when activated. Also Tooltip text for a button on the Source tab. */
 ui_strings.S_BUTTON_LABEL_LOG_THREADS = "Log threads";
 
+/* DESC: Enable reformatting of JavaScript. */
+ui_strings.S_BUTTON_LABEL_REFORMAT_JAVASCRIPT = "Pretty print JavaScript";
```
Comment (hzr) [inline src/ui-strings/ui_strings-en.js]: This string is not new, but I would use "Pretty-print" with a dash.

File: src/ui-strings/ui_strings-en.js [hunk 3]
```diff
@@ -1062,6 +1128,9 @@ ui_strings.S_LABEL_SEARCH_TYPE_XPATH = "XPath";
 /* DESC: Settings label to show a tooltip for the hovered identifier in the source view. */
 ui_strings.S_LABEL_SHOW_JS_TOOLTIP = "Show inspection tooltip";
 
+/* DESC: Enable smart reformatting of JavaScript. */
+ui_strings.S_LABEL_SMART_REFORMAT_JAVASCRIPT = "Smart JavaScript pretty printing";
```
Comment (hzr) [inline src/ui-strings/ui_strings-en.js]: Likewise, a dash here ("pretty-printing").

**Comment by hzr** [comment]
Otherwise fine!

**Comment by hzr** [comment]
Looking fine now.

**Comment by danfooo** [comment]
It's hard to see which strings are actually changed. Maybe they should be sorted on bugfixes too so the diff looks better?

**Comment by chriskr** [comment]
The UI strings have not really changed except for running cleanrepo.

**Comment by danfooo** [comment]
So what's said about "Smart JavaScript pretty printing" above is not introduces in this pull request? The rest looks fine.

**Comment by chriskr** [comment]
Yes, you're right, that is part of https://github.com/operasoftware/dragonfly/commit/e33f8b61368d4f4afed0c7f7c6dead62f360f905. Probably easier to review per commit. " Removed duplicates in ui strings" is basically the result of cleanrepo.

**Pull request merged**